test/commands/iota/configs.test.ts

```diff
@@ -23,14 +24,31 @@ const didKeyWallet = {
 
 const configuration = {
```

Comment:
We could rename the const `configuration` to `configurationWebsocket` to be consistent with `configurationRedirect`.